geometry_msgs/Point[] points
---
bool success
string msg